extensions/extensions-pipeline-invocation-service/src/main/java/com/boozallen/aissemble/pipeline/invocation/service/util/exec/HelmCommandExecutor.java

```diff
@@ -102,8 +104,7 @@ public List<String> createBaseHelmInstallArgs(String appName) {
 
         args.add("install");
         args.add(appName);
-        args.add("--repo");
-        args.add(aissembleHelmRepoUrl);
+        args.add(aissembleHelmRepoProtocol + "://" + aissembleHelmRepoUrl + "/aissemble-spark-application-chart");
```

Comment:
Q: so it seems like this is the main fix? It was just missing the protocol?

Reply:
Yep, this [SO](https://stackoverflowteams.com/c/boozallensolutioncenter/questions/562/563#563) sums it up well